src/app/admin/page.tsx

```diff
@@ -1,21 +1,55 @@
 'use client';
 
-// TODO: Only let admins access this page
+import { isNotEmpty, useForm } from '@mantine/form';
+import { Button, TextInput } from '@mantine/core';
 import { useAuth } from '@/sessions/AuthContext';
 
 export default function AdminPage() {
   const { user } = useAuth();
-  const isAdmin = user && user.customClaims.role === 'admin';
-  if (!user || !isAdmin) {
-    return (
-      <div>
-        <h1 className="text-white">You are not an admin</h1>
-      </div>
-    );
-  }
+
+  const deleteUserForm = useForm({
+    initialValues: {
+      id: '',
+    },
+    validate: {
+      id: isNotEmpty('ID is required'),
+    },
+  });
+
+  const handleDelete = async (values: { id: string }) => {
+    const response = await fetch('/api/users/delete', {
+      method: 'DELETE',
+      headers: {
+        'Content-Type': 'application/json',
+        Authorization: `Bearer ${user?.token}`,
+      },
+      body: JSON.stringify(values),
+    });
+
+    if (response.ok) {
+      console.log('User deleted successfully');
+    } else {
+      console.error('Failed to delete user');
+    }
+  };
+
   return (
     <div>
       <h1 className="text-white">Admin Page</h1>
```

Comment:
i assume this is to just test that it works and we can delete user from admin page but there should be some type of visual indication that the user was deleted successfully after

Reply:
yeah thats just a test for reference to show how the client would call